README.md

```diff
@@ -45,13 +45,13 @@ sudo apt-get install -y curl gnupg
 3. Add the Zuma repository to your system's software repository list:
 
 ```bash
-echo "deb https://apt.abklabs.com/ zuma main" | sudo tee /etc/apt/sources.list.d/zuma.list
+echo "deb https://apt.abklabs.com/zuma dev main" | sudo tee /etc/apt/sources.list.d/zuma.list
 ```
 
 4. Import the repository's GPG key:
 
 ```bash
-curl -s https://apt.abklabs.com/pubkey.gpg | sudo apt-key add -
+curl -s https://apt.abklabs.com/keys/abklabs-archive-dev.asc | sudo apt-key add -
```

Comment:
The use of dev is nice can we use that while we continue to publish the head of the projects? Is that the correct way to think of it.